src/components/Controls.tsx

```diff
@@ -83,7 +87,14 @@ export default function Controls(props: IControlsProps): JSX.Element {
         <Col>
           <h2>notes:</h2>
 
-          <Form.Control as="textarea" rows={20} />
+          <Form.Control
+            as="textarea"
+            rows={20}
+            style={{
+              backgroundColor: dark ? "dimgray" : "white",
+              color: dark ? "white" : " black",
```

Comment:
In the future, we should unify the styling of the textarea with the other "normal" text inputs, which we're currently using for the number of games played and won.

Reply:
yeah it wasn't the best solution. was trying to push the changes out asap.

Kedyn opened a new issue Kedyn/fusliez-notes#13 to discuss about the project structure and how to approach managing styling files and whatnot

I guess once that's settled we can look into unifying things so it isn't all over the place